### DISCUSSION TITLE
本地部署，界面是中文；海外服务器上部署，界面是英文，想请教一下怎么设置显示的语言

### DISCUSSION THREAD

**Discussion by simoonp** [GaiZhenbiao/ChuanhuChatGPT#662] [Q&A] [answered]
- 在本地部署时用的是代码部署，界面显示中文：

![image-20230415103520731](https://raw.githubusercontent.com/simoonp/upgit_picture/main/2023/04/upgit_20230415_1681526120.png)

- 在云服务器上使用的是docker部署方式，服务器地址是美国，加载后界面显示是英文：

![image-20230415103240886](https://raw.githubusercontent.com/simoonp/upgit_picture/main/2023/04/upgit_20230415_1681525961.png)

- 想问一下，应该在那里修改配置，使得云服务器上部署的 Chuanhu Chat 的界面是中文

**Comment by simoonp** [comment, marked as answer]
- 已解决

新版本的 `config.json` 文件中增加了语言选项，默认自动适配英文和日文

将  `config.json` 文件中的 `"language": "auto"` 修改为 `"language": "zh_CN"` ，再重新启动就是中文界面了

**Reply by oldfishoh** [reply]
改了后还是这样

**Reply by ChiefHYK** [reply]
对，设置了还是中文，哈哈

**Reply by Keldos-Li** [reply]
@ChiefHYK 所以你希望设置为何种语言呢？如对语言设置有疑问，请参考：[设置语言](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/使用教程#设置语言)

**Reply by YhYu789** [reply]
为什么设置了config，ip也变换了，语言还是中文呢？

**Reply by GaiZhenbiao** [reply]
在config中设置了之后，需要重启服务
